test/support/data_case.ex

```diff
@@ -49,8 +49,8 @@ defmodule Skate.DataCase do
   """
   def errors_on(changeset) do
     Ecto.Changeset.traverse_errors(changeset, fn {message, opts} ->
-      Enum.reduce(opts, message, fn {key, value}, acc ->
-        String.replace(acc, "%{#{key}}", to_string(value))
+      Regex.replace(~r"%{(\w+)}", message, fn _, key ->
+        opts |> Keyword.get(String.to_existing_atom(key), key) |> to_string()
```

Comment:
https://github.com/phoenixframework/phoenix/blob/main/installer/templates/phx_ecto/data_case.ex#L50-L55

Dunno much about this change, but I don't think we use this function yet.

Reply:
Good enough for me. Thanks for the link!